.gitignore

```diff
@@ -15,7 +15,7 @@ app/app.js
 blockchain_explorer.db.gz
 reports
 .history
-.env
+.env.*
```

Comment:
Please update the entry to ensure that tracking on `!.env.example` isn't lost in the future.

```suggestion
## Local environment files
.env.*
!.env.example
```

Reply:
Fixed